### DISCUSSION TITLE
Scenario "Lisbon": etcd SSL cert troubles

### DISCUSSION THREAD

**Discussion by fduran** [SadServers/sadservers#31] [Scenarios]
Discussion about the scenario [Lisbon](https://github.com/fduran/sadservers/tree/main/scenarios/lisbon)

**Comment by Spikeophant** [comment]
Cert is now expired and you need to issue a new certificate, not sure from looking in the repo how to fix the certificate problem, if you point me in the right direction I'll update this challenge to either have the clues for genning the cert, or just put a new cert in the image.

**Comment by zerotrustprivacy** [comment]
Not seeing any _usr/bin/iptables_ directory

**Reply by rachelsfba** [reply]
The `iptables` binary is in `/usr/sbin`, not `/usr/bin/` (and is a file, not a directory).

**Reply by fduran** [reply]
Right, the [second clue](https://github.com/fduran/sadservers/tree/main/scenarios/lisbon#clues) for this scenario shows `/usr/sbin/iptables`

**Comment by fduran** [comment]
this scenario has been re-built
